libs/domain/picking/api/feature.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  AppFeature,
+  DefaultJsonAPITransformerService,
+  JsonAPITransformerService,
+} from '@spryker-oryx/core';
+import { Provider } from '@spryker-oryx/di';
+import {
+  PickingHttpDefaultService,
+  PickingHttpService,
+  PickingListAdapter,
+  PickingListDefaultAdapter,
+  PickingListDefaultService,
+  PickingListService,
+  WarehouseUserAssignmentsAdapter,
+  WarehouseUserAssignmentsDefaultAdapter,
+  WarehouseUserAssignmentsDefaultService,
+  WarehouseUserAssignmentsService,
+  warehouseUserAssignmentNormalizer,
+  warehouseUserAssignmentsNormalizer,
+} from './services';
+
+export class PickingApiFeature implements AppFeature {
```

Comment:
I don't really like the name `api` for this sub-package, it can be confusing for end-users as for me. Any other suggestions? Maybe `services` as we already using for other packages?

Reply:
Renamed to `picking/services`